api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -477,10 +477,11 @@ public void updateWorkspaceBillingAccount(DbWorkspace workspace, String newBilli
         .billing
         .freeTierBillingAccountNames()
         .contains(newBillingAccountName)) {
+      boolean hasInitialCreditsRemaining =
```

Comment:
I would like update all uses of `.freeTierBillingAccountNames().contains(billingAccountName)` to use `isFreeTier()` instead.  Looks like that's only;
* here
* L451 in this file
* CloudTaskInitialCreditsExpiryController